.github/workflows/build-multi-stage.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: multi-arch-build
+# yamllint disable-line rule:truthy
+on:
+  push:
```

Comment:
Can this be empty? Or should it be removed? What does `yamllint` say about it?

Run `make containerized-build TARGET=lint-yaml`

Reply:
Yes, this can be empty, lint-yaml doesn't complain about it.